src/test/unit/osd_unittest.cc

```diff
@@ -1036,5 +1036,9 @@ extern "C" {
         return false;
     }
 
+    bool isConstantIdleMode(void) {
```

Comment:
`...Active()`?

Reply:
no, because look at the block above, it's "isFlipOverAfterCrashMode". i made it consistent.